libcodechecker/libclient/client.py

```diff
@@ -243,9 +244,8 @@ def setup_client(product_url):
     _, session_token = setup_auth_client(protocol, host, port)
 
     # Check if the product exists.
-    product_client = setup_product_client(protocol, host, port,
-                                          product_name=None)
-    product = product_client.getProducts(product_name, None)
+    client = setup_product_client(protocol, host, port, product_name=None)
```

Comment:
I am a bit confused. We set up product client also if `product_client` is `False`? Maybe the naming is not the best?

Reply:
Yes we set it up because the availability of the product needs to be checked even if a service client is requested.